api/src/main/java/org/openmrs/api/ConditionService.java

```diff
@@ -62,6 +62,17 @@
 	@Authorized({ PrivilegeConstants.GET_CONDITIONS })
 	List<Condition> getActiveConditions(Patient patient) throws APIException;
 
+    /**
+	 * Gets a patient's active and inactive conditions
```

Comment:
I think this description needs to be changed in order to match the current changes.

Reply:
@dkayiwa  I updated the PR. Please take a look :)